pyproject.toml

```diff
@@ -27,12 +27,20 @@ Documentation = "https://github.com/h2oai/authn-py#readme"
 Issues = "https://github.com/h2oai/authn-py/issues"
 Source = "https://github.com/h2oai/authn-py"
 
-[tool.hatch.build.targets.sdist]
-include = ["/src", "/tests", "CHANGELOG.md"]
-
 [tool.hatch.build.targets.wheel]
 packages = ["src/h2o_authn"]
 
+[project.optional-dependencies]
+# TODO(@zoido): Set to released version once available.
+# DO NOT MERGE
```

Comment:
Out of my scope, but this looks scary for a PR?

Reply:
I won't merge this PR before I change the reference to the released version.

Reply:
FIXED.